docs/sources/k6/next/using-k6-browser/recommended-practices/simulate-user-input-delay.md

```diff
@@ -34,13 +34,69 @@ The browser module predominantly provides asynchronous APIs, so it's best to avo
 
 In the browser modules there are various asynchronous APIs that can be used to wait for certain states:
 
-| Method                                           | Description                                                                 |
-| ------------------------------------------------ | --------------------------------------------------------------------------- |
-| [page.waitForFunction](#pagewaitforfunction)     | Waits for the given function to return a truthy value.                      |
-| [page.waitForLoadState](#pagewaitforloadstate)   | Waits for the specified page life cycle event.                              |
-| [page.waitForNavigation](#pagewaitfornavigation) | Waits for the navigation to complete after one starts.                      |
-| [locator.waitFor](#locatorwaitfor)               | Wait for the element to be in a particular state.                           |
-| [page.waitForTimeout](#pagewaitfortimeout)       | Waits the given time. _Use this instead of `sleep` in your frontend tests_. |
+| Method                                                     | Description                                                                 |
+| ---------------------------------------------------------- | --------------------------------------------------------------------------- |
+| [browserContext.waitForEvent](#browsercontextwaitforevent) | Waits for the selected event to fire and returns its value.                 |
+| [page.waitForFunction](#pagewaitforfunction)               | Waits for the given function to return a truthy value.                      |
+| [page.waitForLoadState](#pagewaitforloadstate)             | Waits for the specified page life cycle event.                              |
+| [page.waitForNavigation](#pagewaitfornavigation)           | Waits for the navigation to complete after one starts.                      |
+| [locator.waitFor](#locatorwaitfor)                         | Wait for the element to be in a particular state.                           |
+| [page.waitForTimeout](#pagewaitfortimeout)                 | Waits the given time. _Use this instead of `sleep` in your frontend tests_. |
```

Comment:
Does it make sense to sort these by `type.method`?

Reply:
Not sure I get what you mean. Happy to consider a suggestion 👍

Reply:
```suggestion
| [browserContext.waitForEvent](#browsercontextwaitforevent) | Waits for the selected event to fire and returns its value.                 |
| [page.waitForFunction](#pagewaitforfunction)               | Waits for the given function to return a truthy value.                      |
| [page.waitForLoadState](#pagewaitforloadstate)             | Waits for the specified page life cycle event.                              |
| [page.waitForNavigation](#pagewaitfornavigation)           | Waits for the navigation to complete after one starts.                      |
| [page.waitForTimeout](#pagewaitfortimeout)                 | Waits the given time. _Use this instead of `sleep` in your frontend tests_. |
| [locator.waitFor](#locatorwaitfor)                         | Wait for the element to be in a particular state.                           |
```